src/mapContainer.ts

```diff
@@ -96,6 +159,11 @@ export class MapContainer {
         clusterGroup: leaflet.MarkerClusterGroup;
         /** The markers currently on the map */
         markers: MarkersMap = new Map();
+        /** The vertices currently on the map */
+        vertices: Map<string, Vertex> = new Map();
+        /** The polylines currently on the map */
```

Comment:
I think that beyond communicating with Leaflet.js, we should stick to the term 'edges'